executor/src/main/java/org/k8loud/executor/service/ExecutionWrapperServiceImpl.java

```diff
@@ -24,10 +25,9 @@ public ResponseEntity<String> execute(ExecutionRQ request) {
         //TODO interceptor
         Action action = mapperService.map(request);
         if (action == null) {
-            executionService.execute(action);
-        } else {
             return new ResponseEntity<>("dupa", HttpStatus.BAD_REQUEST);
         }
-        return new ResponseEntity<>("Jest git", HttpStatus.OK);
+        ExecutionRS response = executionService.execute(action);
+        return new ResponseEntity<>("Action response: " + response, HttpStatus.OK);
```

Comment:
Is the "Action response: " needed?

Reply:
I feeling like we should parser this response and create some kind of responses